### PR TITLE
Michael hennecke/daos 5579

### PR DESCRIPTION
**Pull request by Michael-Hennecke**

### CODE REVIEW

**Comment by Michael-Hennecke** [comment]
Hmm, this PR seems to re-apply the earlier commits on this branch, including to daos_server.yml where the ## issue arose with unit testing and had been fixed meanwhile...

**Comment by jolivier23** [comment]
Conflict needs to be resolved.   Also, please update the title and description.  I think this may be the main reason why we use new branches for commits as the initial commit gets used for that purpose in such cases but if you upload a patch for a PR with multiple commits, it looks like this by default.

**Comment by Michael-Hennecke** [comment]
I'll close this PR and clean things up before re-submitting.

File: doc/admin/deployment.md [hunk 1]
```diff
@@ -797,7 +799,10 @@ $ daos_agent -i -o <'path to agent configuration file/daos_agent.yml'> &
 ```
 
 Alternatively, the DAOS Agent can be started as a systemd service. The DAOS Agent
-unit file is installed in the correct location when installing from RPMs.
+unit file is installed in the correct location when installing from RPMs. 
+If you want to run the DAOS Agent without certificates (not recommended in production
+deployments), you need to add the `-i` option to the systemd `ExecStart` invocation
+(see below).
```
[the remark this answers is not in the document]
Comment (Michael-Hennecke) [reply doc/admin/deployment.md:806]: I closed this PR because I messed up the branch. There's a new clean PR #3379 instead.

**Pull request closed without merging**